src/instructlab/configuration.py

```diff
@@ -585,13 +608,154 @@ def ensure_storage_directories_exist():
         "lora_target_modules": ["q_proj", "k_proj", "v_proj", "o_proj"],
     }
 
+    FOUR_AND_EIGHT_GPU_TRAIN_AH = _train(
+        additional_args={
+            "warmup_steps": 10,
+            "learning_rate": 2e-5,
+            "lora_dropout": 0.1,
+            "lora_alpha": 32,
+        },
+        ckpt_output_dir=os.path.join(DEFAULTS._data_dir, "checkpoints"),
+        data_output_dir=os.path.join(DEFAULTS._data_dir, "internal"),
+        data_path=os.path.join(DEFAULTS._data_dir, "datasets"),
+        nproc_per_node=4,
```

Comment:
Why are we using only 4 procs per node when we have 8 GPUs? Should this be 8 instead in that case?